website/project/model.py

```diff
@@ -2640,7 +2640,7 @@ def to_json(self):
             "key": self.key,
             "name": self.name,
             "creator": {'fullname': self.creator.fullname, 'url': self.creator.profile_url},
-            "nodes": [{'title': x.title, 'url': x.url, 'scale': str(self.node_scale(x)) + 'px', 'imgUrl': self.node_icon(x)}
+            "nodes": [{'title': x.title, 'url': x.url, 'scale': str(self.node_scale(x)) + 'px', 'imgUrl': self.node_icon(x), 'category': x.category}
```

Comment:
When I removed that the formatting was messed up but I can look into why (possibly a CSS issue).